src/ripple/app/tests/TxQ_test.cpp

```diff
@@ -1562,6 +1613,23 @@ class TxQ_test : public beast::unit_test::suite
                 expect(result.isMember(jss::current_ledger_size));
                 expect(result.isMember(jss::current_queue_size));
                 expect(result.isMember(jss::expected_ledger_size));
+                expect(!result.isMember(jss::max_queue_size));
+            }
+
+            env.close();
+
+            fee = env.rpc("fee");
+
+            if (expect(fee.isMember(jss::result) &&
+                !RPC::contains_error(fee[jss::result])))
+            {
+                auto const& result = fee[jss::result];
+                expect(result.isMember(jss::drops));
```

Comment:
Possibly expect the following to be present in the drops field
      base_fee, median_fee, minimum_fee, open_ledger_fee
and in 'levels'
      median_level, minimum_level, open_ledger_level, reference_level